proto/rill/local/v1/api.proto

```diff
@@ -103,6 +105,7 @@ message DeployProjectResponse {
 message RedeployProjectRequest {
```

Comment:
Given we now have project_id we should remove `org` and `project_name` ?

Reply:
True. I assumed it was used elsewhere. I guess it is not.